config/jobs/cadvisor/OWNERS

```diff
@@ -1,10 +1,10 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 reviewers:
+- sig-node-test-reviewers
```

Comment:
This seems to be scope escalation. I would probably not couple sig-node-test-reviewers with this as they may not have much familar with cadvisor code base.

Can we leave this one as is for now? Or add a new ALIASE for cadvisor?

Reply:
I still thing sig node test approvers should be able to control these jobs. Removed reviewers

Reply:
With the lack of good tooling, managing SIG Node OWNER files would require a part time admin to keep everything in-tact if we list individuals in separate files.